src/App.tsx

```diff
@@ -1,24 +1,164 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Notification } from './components/Notification';
+import { Header } from './components/Header';
 import { UserWarning } from './UserWarning';
+import { addTodos, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorNotice } from './types/ErrorNotice';
+import { Filter } from './types/Filter';
+import { getVisibleTodos } from './helper/getVisibleTodos';
 
-const USER_ID = 0;
+const USER_ID = 6762;
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodos] = useState<Todo[]>([]);
+  const [hasError, setError] = useState(false);
+  const [errorMessage, setMessageError] = useState('');
```

Comment:
you don't need 2 error states. `hasError` can be calculated
```suggestion
  const [errorMessage, setMessageError] = useState('');
  const hasError = !!errorMessage || errorMessage !== '';
```